core/src/components/range/test/a11y/range.e2e.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('range: a11y', () => {
+  test('should not have visual regressions', async ({ page, skip, browserName }) => {
+    skip.rtl();
+    skip.mode('ios', 'iOS mode does not display hover/active/focus styles.');
+
+    await page.setContent(
+      `<ion-app>
+        <ion-content>
+          <ion-range min="0" max="100" value="80"></ion-range>
+        </ion-content>
+      </ion-app>
+        `
+    );
+
+    const tabKey = browserName === 'webkit' ? 'Alt+Tab' : 'Tab';
+
+    const range = page.locator('ion-range');
+    const rangeHandle = page.locator('ion-range .range-knob-handle');
+
+    await page.keyboard.press(tabKey);
+    await page.waitForChanges();
+
+    await rangeHandle.hover();
+    await page.waitForChanges();
+
+    expect(await range.screenshot()).toMatchSnapshot(`range-hover-${page.getSnapshotSettings()}.png`);
+
+    await page.keyboard.press(tabKey);
+    await page.waitForChanges();
+
+    expect(await range.screenshot()).toMatchSnapshot(`range-focus-${page.getSnapshotSettings()}.png`);
+
+    await rangeHandle.click();
```

Comment:
Since the active styling only shows when you're actively dragging the handle, should this be a mousedown instead?